connaisseur/constants.py

```diff
@@ -1 +1,2 @@
+AIO_TIMEOUT_SECONDS = 29
```

Comment:
think we could go closer to 30s? Ultimately, we'll return even if k8s timed out. It's just a bit unfortunate to loose a full second

Reply:
https://docs.aiohttp.org/en/stable/client_quickstart.html#timeouts

> aiohttp ceils timeout if the value is equal or greater than 5 seconds. The timeout expires at the next integer second greater than current_time + timeout.

Which means 29 will on average be 29,5s already and might be 29.9999 (in which case we'll probably not return in time. We could go down to 28 to fix that, but then again I agree with you on the point that we shouldn't "waste" too much time). If we went to 29,5, we'd basically always fail to return in time

Reply:
good point. thanks for clarifying

Reply:
Think I'll add a comment outlining this. Thanks for asking